slash/react/src/Form/Checkbox/CheckboxInput.tsx

```diff
@@ -59,6 +61,8 @@ const CheckboxInput = forwardRef<HTMLInputElement, Props>(
               : classModifier
           }
           ref={inputRef}
+          aria-describedby={errorUseId}
+          aria-invalid={!!(message && forceDisplayMessage)}
```

Comment:
l'utilisation du `!!` est déconseillé car c'est peu clair sur l'intention. Pour transformer qq chose en bool, la maniere la plus claire est de faire 
```suggestion
          aria-invalid={Boolean(message && forceDisplayMessage)}
```